Catalog/Shared/Pickers/PickerStyles.swift

```diff
@@ -0,0 +1,133 @@
+//
+//  PickerStyles.swift
+//  Catalog (iOS)
+//
+//  Created by Djuro on 1/5/22.
+//
+
+import Props
+import SwiftUI
+
+extension PropGroup {
+    static let systemPickers = PropGroup(name: "System pickers", samples: [
+        DefaultPickerSample(),
+        InlinePickerSample(),
+        MenuPickerSample(),
+        SegmentedPickerSample(),
+        WheelPickerSample(),
+    ])
+}
+
+struct DefaultPickerSample: PropSampleable {
+    let name = "Default picker style"
+    let notes: String? = ".pickerStyle(DefaultPickerStyle())"
+    let source = "System"
+    let keywords = "picker system default"
+    @ViewBuilder var body: AnyView {
+        AnyView(
+            Picker("My Picker", selection: .constant("Soccer ⚽️")) {
+                Text("Soccer ⚽️")
+                    .tag(0)
+                Text("Basketball 🏀")
+                    .tag(1)
+                Text("Baseball ⚾️")
+                    .tag(2)
+            }
+            .pickerStyle(DefaultPickerStyle())
+        )
+    }
+}
+
+struct InlinePickerSample: PropSampleable {
+    let name = "Inline picker style"
+    let notes: String? = ".pickerStyle(InlinePickerStyle())"
+    let source = "System"
+    let keywords = "picker system inline"
+    @ViewBuilder var body: AnyView {
+        AnyView(
+            Picker("My Picker", selection: .constant("Soccer ⚽️")) {
+                Text("Soccer ⚽️")
+                    .tag(0)
+                Text("Basketball 🏀")
+                    .tag(1)
+                Text("Baseball ⚾️")
+                    .tag(2)
+            }
+            .pickerStyle(InlinePickerStyle())
```

Comment:
Can you change all of these to the `.inline` style, please? And on the other PRs, too. If they exist, we should use them. Also in the `notes` string for each one.